core/src/main/scala/org/bitcoins/core/protocol/dlc/CETCalculator.scala

```diff
@@ -13,6 +13,30 @@ import scala.annotation.tailrec
 
 object CETCalculator {
 
+  /** A Vector of digits in a given base.
+    *
+    * For example if we are using base 2 (binary)
+    * This could look like
+    * Vector(0,1,0,1,0,1,1,1,0)
+    *
+    * or if we are using base 16 (hex)
+    * Vector(9,8,10,11,2,3,2,4,1,15)
+    */
+  type Digits = Vector[Int]
+
+  /** A Vector of Digits corresponding to each oracle in order */
+  case class MultiOracleDigits(vec: Vector[Digits]) {
+    def toVector: Vector[Digits] = vec
+  }
```

Comment:
I think I prefer this as a type alias after having looked at the code but everything else looks good imo

Reply:
If you are indifferent I would really like to keep this. I find it frustrating to work with type aliases in part due to the scala compiler errors. With type aliases on type aliases (`Digits` inside of `MultiOracleDigits`) you can get error messages that look like this 

```scala
[error]  required: org.bitcoins.core.protocol.dlc.CETCalculator.MultiOracleDigits
[error]     (which expands to)  scala.collection.immutable.Vector[scala.collection.immutable.Vector[Int]]
[error]         builder.++=(Vector(1))
```

as compared to what I have on c15cbde5478c1b14fa932353ea886d8e6bb6cc64

```scala
[error] /home/chris/dev/bitcoin-s/core/src/main/scala/org/bitcoins/core/protocol/dlc/CETCalculator.scala:718:28: type mismatch;
[error]  found   : scala.collection.immutable.Vector[Int]
[error]  required: org.bitcoins.core.protocol.dlc.CETCalculator.MultiOracleDigits
[error]           builder.+=(Vector(1))
```

I find it hard to reason about type aliases inside of type aliases, but maybe this is just a personal preference. If you really want me to switch it back I will.

Reply:
Hm, between those two error messages I find the first much more informative? It is literally the second one but with even more extra information?

Reply:
Yes, that's the problem. I find it to be a wall of text that includes unnecessary information if I'm just trying to get things to compile (the type I care about is `MultiOracleDigits`, not trying to grok that the implementation of `MultiOracleDigits` is a `Vector[Vector[Int]]`.

This case isn't really that bad, but if you go hog wild with type aliases like the scala slick project or previous iterations of suredbits-api, I find type aliases get expanded into walls of text by the scala compiler that are really hard to understand what is going on and maintain. Maybe scala just has a poor implementation of them, but it's the tool we are using

Any way, this case seems to be confined and we aren't going to agree so I will change the code for the sake of productivity.